include/nuttx/signal.h

```diff
@@ -191,6 +204,54 @@ int nxsig_addset(FAR sigset_t *set, int signo);
 
 int nxsig_delset(FAR sigset_t *set, int signo);
 
+/****************************************************************************
+ * Name: nxsig_nandset
+ *
+ * Description:
+ *   This function returns the intersection of the left set and the
+ *   complement of the right set in dest.
+ *
+ * Input Parameters:
+ *   dest  - The location to return the result
+ *   left - The uncomplemented set used in the intersection
+ *   right  - The set that will be omplemented and used in the intersection
```

Comment:
```suggestion
 *   dest  - The location to return the result
 *   left  - The uncomplemented set used in the intersection
 *   right - The set that will be complemented and used in the intersection
```

Reply:
Also let's try to avoid `to return the result` and change it to `to store the result` as API does not return pointer to `sigset_t`, but an error code.

Reply:
Done. Thanks!